pinot-common/src/main/java/org/apache/pinot/common/function/TransformFunctionType.java

```diff
@@ -125,7 +125,10 @@ public enum TransformFunctionType {
   ARRAYMAX("arrayMax"),
   ARRAYSUM("arraySum"),
   VALUEIN("valueIn"),
-  MAPVALUE("mapValue"),
+  MAPVALUE("mapValue", ReturnTypes.cascade(opBinding ->
```

Comment:
ARG2 is an array of X, X={INT, STRING, ...}, the return table should be this X.

Reply:
you meant `X = {INT | STRING | ...}` correct?

Reply:
yes